docs/cugraph/source/installation/source_build.md

```diff
@@ -61,7 +61,7 @@ conda env update --name cugraph_dev --file $CUGRAPH_HOME/conda/environments/all_
 conda activate cugraph_dev
 
 # for CUDA 12.x
-conda env update --name cugraph_dev --file $CUGRAPH_HOME/conda/environments/all_cuda-122_arch-x86_64.yaml
+conda env update --name cugraph_dev --file $CUGRAPH_HOME/conda/environments/all_cuda-125_arch-x86_64.yaml
```

Comment:
There's one more right above this that also needs to be fixed:

https://github.com/rapidsai/cugraph/blob/5cdef4ef7f0df564fd85088c86e0e961958c5ef5/docs/cugraph/source/installation/source_build.md#L46

Looks like `rapids-reviser` wouldn't have caught it because it was (incorrectly) using `-12.2` instead of `-122` (so pointing to a file that doesn't exist).